funnel/templates/space.html

```diff
@@ -42,6 +42,17 @@
                 {%- if proposal.user != proposal.speaker %}
                   <small>via {{ proposal.user.pickername }}</small>
                 {%- endif %}
+                {%- if g.user and 'edit-space' in g.permissions %}
+                  <span class='outstation-speaker'>
+                    {%- if proposal.is_outstation_speaker(space.location_geonameid) %}
```

Comment:
Don't bother about unknown locations. You only want to mark when the speaker's location is not matching the event's location.